app/views/admin/feedback_messages/_feedback_message.html.erb

```diff
@@ -145,7 +145,7 @@
               <%= text_area_tag :affected_email_body, affected_email_details[:body], class: "form-control my-1", style: "height: 300px;", id: "affected__body__#{feedback_message.id}" %>
             </div>
           </div>
-          <button class="btn btn-primary" type="button" id="send__email__btn__<%= feedback_message.id %>">Send Email ✉️</button>
+          <button class="btn btn-primary" type="button" id="send__email__btn__<%= feedback_message.id %>" data-path="<%= send_email_admin_reports_path %>">Send Email ✉️</button>
```

Comment:
Wanted to double check that this pattern seems like a feasible one?

Reply:
Seems a bit odd to me, shouldn't there be a `<form>` with this path?

Reply:
Hmm yeah that would make sense, I went into `lets move everything to path helpers mode` and didnt think of that 😅. Let me see why we didn't use a form

Reply:
@rhymes I tried moving this out into a form, and boy oh boy has things been breaking. The `send email` one is particularly complicated and we also do a whole lot of updates in these methods using javascript. I know that this is an odd pattern, but if reviewers don't think its too much of an issue, I'd like to merge it and add an issue to the engineering board to refactor this templates to use forms.

Reply:
Instead of adding it to a data-attribute here and then using it as the path for each `fetch`, given that the JS is inline the ERB file, how about we directly render the `send_email_admin_reports_path` (and others) in the JS section below? This way we avoid the extra step of querying for the element to get the path for each one.

Refactoring this logic to remove the JS and make these rely on `<form>` instead (as suggested by @rhymes) would be great, but it does feel like a bit outside the scope for this PR and making that a separate issue sounds reasonable to me.

As a side note: I always forget about [button_to](https://api.rubyonrails.org/v5.2.3/classes/ActionView/Helpers/UrlHelper.html#method-i-button_to) and it feels like a great options for these

Reply:
Apologies for my bad code here 😅 I think it's fine to refactor in a future PR.

Also shaking my head at my former self for writing a script that RENDERS IN EACH PARTIAL 🤦‍♂️ 🤦‍♂️ 🤦‍♂️